optionlab/strategies.py

```diff
@@ -0,0 +1,187 @@
+import datetime as dt
+from typing import Literal
+
+from optionlab.models import Strategy, StockStrategy, OptionStrategy
+
+# TODO: vet and add to this list
+NamedStrategy = Literal[
+    "covered-call", "married-put", "bull-call", "bear-put", "protective-collar"
+]
+
+
+def generate_strategies(
```

Comment:
This function name is misleading. And the most common options strategies are missing. Iron condor, butterfly, strangle, straddle, naked call, naked put are some that come to my mind.